c-dependencies/js-compute-runtime/Makefile

```diff
@@ -53,8 +58,33 @@ $(RUST_URL_LIB): $(RUST_URL_RS_FILES) $(RUST_URL_SRC)/Cargo.toml $(RUST_URL_SRC)
 	$(WASI_CXX) $(CXX_FLAGS) $(CXX_OPT) $(DEFINES) -I $(SM_SRC)include -MMD -MP -c -o $@ $<
 
 js-compute-runtime.wasm: $(FSM_OBJ) $(SM_OBJ) $(RUST_URL_LIB)
-	$(WASI_CXX) $(CXX_FLAGS) $(CXX_OPT) $(DEFINES) $(LD_FLAGS) -o $@ $^
-	$(WASM_STRIP)
+	$(WASI_CXX) $(CXX_FLAGS) $(CXX_OPT) $(DEFINES) $(LD_FLAGS) -o $@ $^ && \
+		$(call WASM_STRIP,$@)
 
 initialized-js-compute-runtime.wasm: js-compute-runtime.wasm $(INIT_JS)
 	cat $(INIT_JS) | $(WIZER) --allow-wasi --dir=. -r _start=wizer.resume -o $@ $<
+
+clean:
+	$(RM) compile_commands.json $(FSM_OBJ)
+
+distclean: clean
+	$(RM) $(FSM_DEP) compiler_flags
+
+# technically this depends on the value of the FSM_CPP variable
```

Comment:
I think you could safely add `$(FSM_CPP)` to the dependencies of this rule to express what this comment says.

Reply:
I didn't want it to depend on the files, only the state of that variable. I could pipe them into the `compiler_flags` file as well though...

Reply:
Oh right, I get it. Well, maybe the rule runs fast enough that you could mark it as `.PHONY` and not worry about its dependencies?

Reply:
Entirely reasonable! I'll make that change.